x-pack/plugins/infra/public/containers/waffle/waffle_nodes.gql_query.ts

```diff
@@ -6,17 +6,17 @@
 
 import gql from 'graphql-tag';
 
-export const mapQuery = gql`
-  query MapQuery(
+export const waffleNodesQuery = gql`
+  query WaffleNodesQuery(
     $id: ID!
     $timerange: InfraTimerangeInput!
-    $filters: [InfraFilterInput!]
+    $filterQuery: String
     $metrics: [InfraMetricInput!]
     $path: [InfraPathInput!]
   ) {
     source(id: $id) {
       id
-      map(timerange: $timerange, filters: $filters) {
+      map(timerange: $timerange, filterQuery: $filterQuery) {
```

Comment:
I really like that you renamed `Map` to `Waffle*` in most (all?) other places because it's more descriptive, would it be a lot of effort to rename this endpoint as well? Not necessarily in this PR.

Reply:
yes, I would like that too... I'll create a separate PR though. I was thinking of calling it something like `nodes` because that's what it returns and because it will be used by other parts of the ui too (e.g. by a potential table view). Is that still too generic?

Reply:
`nodes` sounds good to me.